crates/bitwarden/src/platform/fido2/authenticator.rs

```diff
@@ -51,24 +58,19 @@ impl<'a> Fido2Authenticator<'a> {
                     .pub_key_cred_params
                     .into_iter()
                     .map(TryInto::try_into)
-                    .collect::<Result<Vec<_>, _>>()?,
+                    .collect::<Result<_, _>>()?,
                 exclude_list: request
                     .exclude_list
-                    .map(|x| {
-                        x.into_iter()
-                            .map(TryInto::try_into)
-                            .collect::<Result<Vec<_>, _>>()
-                    })
+                    .map(|x| x.into_iter().map(TryInto::try_into).collect())
                     .transpose()?,
                 extensions: request
                     .extensions
                     .map(|e| serde_json::from_str(&e))
                     .transpose()?,
-                // TODO(Fido2): Where do we get these from?
                 options: passkey::types::ctap2::make_credential::Options {
-                    rk: true,
+                    rk: request.require_resident_key,
```

Comment:
shouldn't this be `rk: request.options.rk` now that we added the new options ?

Reply:
Ah yeah good point! I didn't see that the field got deleted when we added the Options, it's changed now.